app/src/main/java/com/cmput301f20t21/bookfriends/services/UserService.java

```diff
@@ -49,4 +48,13 @@ public static UserService getInstance() {
                 .whereLessThan("username", username.concat("~")) // ascii ~ is way larger than 'z'
                 .limit(10).get();
     }
+
+    public Task<Void> updateUser(String email){
```

Comment:
suggestion: refactor/rename to `updateUserEmail` as we are only updating email here.